samples/hello-js/build.gradle.kts

```diff
@@ -48,6 +48,49 @@ tasks {
         }
         into("$buildDir/web")
     }
+
+    val populateNodeModules by registering {
+        group = "build"
+        description = "Populate Node modules directory for karma"
+        val outputDir = file("$buildDir/$name")
+        inputs.property("testCompileClasspath", configurations.testCompileClasspath.get())
+        outputs.dir(outputDir)
+        doLast {
+            val fromJars = configurations.testCompileClasspath.get()
+                .filter { it.name.matches(Regex(".*\\.jar")) }
+                .map { zipTree(it) }
+            copy {
+                includeEmptyDirs = false
+                from(fromJars)
+                into(outputDir)
+                include("**/*.js")
+                exclude("META-INF/**")
+            }
+        }
+    }
+
+    node {
+        version = "10.16.0"
+    }
+
+    karma {
```

Comment:
:x: this is also a project extension, please move its configuration out of the `tasks {}` block